scripts/sync_latest_version.sh

```diff
@@ -37,7 +37,9 @@ function uniq_major_minor_filter() {
 function main() {
     local all_release_tags releases doc_versions
     all_release_tags="$(git tag -l v*.*)"
-    releases="$(uniq_major_minor_filter "${all_release_tags}")"
+    # sed trims this list to 10.
+    all_supported_release_tags="$(echo "${all_release_tags}" | sort -Vr | cut -d "." -f1-2 | sed '1,10!d')"
```

Comment:
Just to confirm here, this would give us `2.10.0` to `2.19.0`?

Reply:
Merged in #3203 into this branch.  As discussed offline, the desired state we want is:
```
releases
v2.20 v2.19 v2.18 v2.17 v2.16 v2.15 v2.14 v2.13 v2.12 v2.11
--------------
doc_versions
v2.10 v2.11 v2.12 v2.13 v2.14 v2.15 v2.16 v2.17 v2.18 v2.19 v2.9
```